files/default/yum_opsmatic_public.repo

```diff
@@ -2,9 +2,9 @@
 name=opsmatic_public
 baseurl=https://packagecloud.io/opsmatic/public/el/6/$basearch
 enabled=1
-gpgcheck=0
+gpgcheck=1
 sslverify=1
-repo_gpgcheck=1
+#repo_gpgcheck=1
```

Comment:
@rabdill any reason for turning off repo-level metadata checks? You guys use an internal repo anyway, so this file doesn't affect you specifically, right?